sdcm/nemesis.py

```diff
@@ -4960,6 +4962,82 @@ def are_gate_closed_messages_raised(log_reader):
         assert not gate_closed_appearing, \
             "After re-enabling binary and gossip, 'gate closed' messages continue to appear"
 
+    def disrupt_refusing_connection_from_banned_node(self, use_iptables=False):
+        """Banned node could not connect with rest nodes in cluster
+
+        If node was removed from cluster for any reason, even if removed node
+        become alive and try to communicate with rest node in cluster, all connections
+        from it should be refused by other nodes in cluster
+        1. on target node block any scylladb process
+        1.1 Pause process
+        1.2 Block with iptables port 9100/10000
+        2. Wait and remove target node from cluster
+        3. start scylla on target node
+        4. Create exclusive connection to target node
+        5. Execute cql command on target node and validate that no operation
```

Comment:
How is executing cql connection from a node related to the fact it was removed from the cluster?

Reply:
Scylla was paused on the node, and after the node was removed from cluster, scylla was starting again. The removed node still think that it is in the cluster. But the cluster doesn't know about it and closed all connections from it. So when run any cql command on the removed node, it executes it and try to replicate to rest nodes on cluster. But cluster have to  ban any connection from this node

Reply:
Ok, but:
Do you check what is the actual failure?
It doesn't look so, in theory the cql port should be opened, but it can be down and the failure will still be considered as failed, isn't it?

I'll comment also in the actual step.

Reply:
@roydahan , So you suggest to add specific Exceptions or add a check that port on banned node is opened? Do i correctly understand you?